dev-guide/cluster-version-operator/dev/clusteroperator.md

```diff
@@ -57,8 +56,20 @@ status:
     - name: operator
       # The string "0.0.1-snapshot" is substituted in the manifests when the payload is built
       version: "0.0.1-snapshot" 
+    - name: operator-image
```

Comment:
I think I can start to create a script to check the changes, the script will:
1. read all operator manifests and check if they have added these entries (at least `operator-image`) in their manifest
2. read `image-references` to see if above operators have values of  <operator-tag-name> in `spec.tags[name=operator]`

Reply:
Are you talking about the test plan? What is the goal of the script?

Reply:
The goal of the script is to check if the real image payload exists in image-references, then I can verify it in advance.
Ignore my test plan here. I just want to know if we can check image-references without upgrade cluster.

Reply:
You can check `ClusterOperator.status.versions[name=operator-image]`.
At the moment, only `co/machine-config` has it if it's version includes https://github.com/openshift/machine-config-operator/pull/4637